loki.yml

```diff
@@ -39,7 +39,10 @@ schema_config:
   - from: 2018-04-15
     # Which store to use for the index.
     store: boltdb
-    object_store: filesystem
+    # this is for storing the log chunks locally
+    #object_store: filesystem
+    # uncomment bellow to store the chunks in S3
+    object_store: s3
```

Comment:
This needs to be changed back to `object_store: filesystem`

Reply:
reveted.